modules/oe_theme_content_project/oe_theme_content_project.post_update.php

```diff
@@ -54,3 +54,15 @@ function oe_theme_content_project_post_update_00002(): void {
   $display = $storage->createFromStorageRecord($display_values);
   $display->save();
 }
+
+/**
+ * Updates the budget field's 'thousand' separator to space.
+ */
+function oe_theme_content_project_post_update_00003(): void {
+  foreach (['teaser', 'full'] as $display) {
+    $view_display = EntityViewDisplay::load("node.oe_project.$display");
+    $view_display->getComponent('oe_project_budget');
+    $component['settings']['thousand_separator'] = ' ';
+    $view_display->setComponent('oe_project_budget', $component)->save();
```

Comment:
This is not good.
We need to update this separately. 
`oe_project_budget` is hidden on teaser view mode so there is no point for updating the `thousand_separator` setting.

Address should show the Country separated by comma only on teaser view mode. 

Could you plase fix the update path?

Reply:
Sorry bad conflict resolve.